### DISCUSSION TITLE
404 - Unknown commit object for CVE-2023-5678

### DISCUSSION THREAD

**Discussion by ganl** [openssl/openssl#22740] [Q&A] [answered]
Fixed in OpenSSL 1.1.1x (git commit) (Affected since 1.1.1)

**https://git.openssl.org/gitweb/?p=openssl.git;a=commitdiff;h=710fee740904b6290fef0dd5536fbcedbc38ff0c**
![image](https://github.com/openssl/openssl/assets/11937688/0f6ea390-406c-4fd0-9da3-fdb4506dc684)

https://www.openssl.org/news/secadv/20231106.txt

Due to the low severity of this issue we are not issuing new releases of
OpenSSL at this time. The fix will be included in the next releases when they
become available.
The fix is also available in commit ddeb4b6c6d527e54ce9a99cba785c0f7776e54b6
(for 3.1) and commit db925ae2e65d0d925adef429afc37f75bd1c2017 (for 3.0).
It is available to premium support customers in commit
**710fee740904b6290fef0dd5536fbcedbc38ff0c** (for 1.1.1) and in commit
34efaef6c103d636ab507a0cc34dca4d3aecc055 (for 1.0.2).

**Comment by nhorman** [comment, marked as answer]
Commits 710fee740904b6290fef0dd5536fbcedbc38ff0c and 34efaef6c103d636ab507a0cc34dca4d3aecc055 reference this fix in the premium support repository, which is not exposed via gitweb

In the public repository this has been fixed on the master branch as commits ddeb4b6 and  db925ae.  It has and will not be fixed in the 1.1.1 and 1.0.2 branches of the public repository as they are out of support.

**Comment by CaptainJ93** [comment]
It looks like this issue has been upgraded to HIGH, and it is blocking our image scan.
https://www.cvedetails.com/cve/CVE-2023-5678/
Can anyone please tell me when and which version will this be fixed in the public repo?

**Comment by mattcaswell** [comment]
> It looks like this issue has been upgraded to HIGH

It continues to be rated as LOW severity by the OpenSSL project.

> Can anyone please tell me when and which version will this be fixed in the public repo?

I refer you to @nhorman's answer above (nit: the commits referred to are for the 3.1 and 3.0 fixes - not the master branch). The fixes will be made available in 3.1.5 and 3.0.13 when they are released.

Fixes for 1.1.1/1.0.2 are available to premium support customers only. See:

https://www.openssl.org/support/contracts.html
